### DISCUSSION TITLE
Are you planning to add view more in Steps component

### DISCUSSION THREAD

**Discussion by KoushikMVN** [ant-design/ant-design-mobile#4252] [Ideas]
I am working on steps component but sometime i have lots of ```Step``` in ```Steps``` component. But i need a view details button or icon for view more step. 

Suppose i have 20 ```Step```  in ```Steps``` component but i want to show only 1st 2 ```Step``` and then when i click on view more or view more icon then all 20 ```Step``` will be shown. 

Is it possible to add pops in ```Steps``` for view more and view less?

**Comment by awmleer** [comment]
Thanks for your proposal. We'll consider about it recently.
